main/main.go

```diff
@@ -17,15 +15,26 @@ import (
 )
 
 var (
-	version            = "1.0.0.0"
-	extensionShortName = "GATestExt"
+	versionMajor    = "1"
+	versionMinor    = "0"
+	versionBuild    = "0"
+	versionRevision = "2"
```

Comment:
You shouldn't bump up version if there's no release. 1 release can have many features, we should send another PR just for version bump whenever we're releasing.
Also, this version should always point to the last released version which was 1.0.0.1 in our case